tests/Testcontainers.Tests/Fixtures/Containers/Unix/DockerMTlsFixture.cs

```diff
@@ -0,0 +1,27 @@
+﻿namespace DotNet.Testcontainers.Tests.Fixtures
+{
+  using System.Collections.Generic;
+  using DotNet.Testcontainers.Builders;
+  using DotNet.Testcontainers.Containers;
+  using JetBrains.Annotations;
+
+  [UsedImplicitly]
+  public sealed class DockerMTlsFixture : ProtectDockerDaemonSocket
+  {
+    public DockerMTlsFixture()
+      : base(new TestcontainersBuilder<TestcontainersContainer>())
+    {
+    }
+
+    public override IList<string> CustomProperties
+    {
+      get
+      {
+        var customProperties = base.CustomProperties;
+        customProperties.Add("docker.tls=true");
```

Comment:
I would suggest to set `docker.tls=false` because client should have automatic handling this to `true` based on `TlsVerify` only (based on docs).

Reply:
Thats a good point. Then we can change it to `customConfiguration.GetDockerTls() || customConfiguration.GetDockerTlsVerify()`

Reply:
It is now `false`